content/html/encyclopedia/semantic/footer.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: "<footer>"
+Subjects:
+  - "Web Development"
+  - "Web Design"
+Tags:
+  - "Semantic"
+  - "Tags"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-html"
+  - "https://www.codecademy.com/learn/paths/web-development"
+---
+
+## Definition 
+
+The `<footer>` semantic tag represents a part of a page which is meant to be at the end of a completed block of content. If it is at the end of an entire page it might include copyright information for the page or additional links to relevant pages, and if it is at the end of an article it might include information about the author.
+
+## Syntax
+
+```html
+<footer>
+  <!-- Footer content goes inside the tag. -->
+</footer>
+``` 
+
+`<footer>` is a tag that wraps around a block of HTML giving it semantic meaning. It has no special attributes, and by default will act similarly to a `<div>`.
+
+## Examples
+
+### Website Footer
+
+After the main content is shown to the users of Bongo Ltd., they should see navigation links to the pages that aren't featured in the header as well as copyright information for the current page.
+
+```html
+<main>
+  <!-- Site content goes here. -->
```

Comment:
Let's take out all the periods in the comments.

Reply:
Ok!